app/src/main/res/values/strings.xml

```diff
@@ -286,79 +288,79 @@
 
     <string name="pref_name_storage_dbPath">Database location</string>
     <string name="pref_name_storage_dbPath_internalStorage">Internal storage</string>
     <string name="pref_name_storage_dbPath_externalStorage">External storage</string>
     <string name="pref_name_storage_dbPath_dbMoved">Database moved</string>
 
-    <string name="pref_categoryName_miscellaneous">Miscellaneous</string>
+    <string name="pref_categoryName_miscellaneous">Misc</string>
     <string name="pref_name_misc_appendWallabagMention">Append wallabag mention</string>
     <string name="pref_desc_misc_appendWallabagMention">Append \"via @wallabagapp\" to shared texts</string>
     <string name="pref_name_misc_handleHttpScheme">Handle HTTP scheme</string>
-    <string name="pref_desc_misc_handleHttpScheme">Show wallabag in the list of web browsers</string>
+    <string name="pref_desc_misc_handleHttpScheme">Show wallabag in the list of web-browsers</string>
     <string name="pref_name_misc_wipeDB">Wipe Database</string>
-    <string name="pref_desc_misc_wipeDB">Wipes all the articles from local database, also removes all not synchronized local changes and URLs</string>
+    <string name="pref_desc_misc_wipeDB">Deletes articles and all unsynced local changes and URLs from the local database.</string>
     <string name="pref_name_misc_wipeDB_confirmTitle">Wipe Database?</string>
-    <string name="pref_name_misc_wipeDB_confirmMessage">Are you sure you want to wipe the database?</string>
+    <string name="pref_name_misc_wipeDB_confirmMessage">Are you sure you want to remove the database?</string>
     <string name="pref_categoryName_misc_localQueue">Local changes</string>
-    <string name="pref_name_misc_localQueue_notice">WARNING</string>
+    <string name="pref_name_misc_localQueue_notice">Warning</string>
     <string name="pref_desc_misc_localQueue_notice">Don\'t do anything here unless you know what you\'re doing</string>
     <string name="pref_name_misc_localQueue_dumpToFile">Dump to file</string>
-    <string name="pref_desc_misc_localQueue_dumpToFile">Dumps all local changes to a txt file (you can study it yourself or attach it to a bug report)</string>
+    <string name="pref_desc_misc_localQueue_dumpToFile">Dumps all local changes to a TXT file (to study yourself or attach to a bug-report)</string>
     <string name="pref_name_misc_localQueue_removeFirstItem">Remove first local queue item</string>
-    <string name="pref_desc_misc_localQueue_removeFirstItem">Removes the first not synchronized item. May help if you get repeating error notifications during sync. Be sure to dump all local changes to file first</string>
+    <string name="pref_desc_misc_localQueue_removeFirstItem">Removes the first unsynced item. May help with repeating errors during notifications syncing. Tip: Save all local changes to a file first.</string>
```

Comment:
It may help with repeating notifications about sync errors.

Is a trailing period necessary if there are multiple sentences?

Reply:
It is just so GNOME to not do it. :) The problem being some don't and some do in translation, and that fails the true positive count for the "trailing dot check".